faithcomesbyhearing/dbp
# FCBHDBP-418 Issue related the endpoint to store a playlist item

# Description
It has fixed related to endpoint to store a playlist item. The endpoint was not returning the data related with the new item stored rather, it was always returning data related with first item of the playlist.

We have used a transaction to guarantee that we can fetch the last playlist item records inserted.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-418

## How Do I QA This
- You must run the following postman URL:
https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/request/12519377-041aa795-75e3-4fb8-bb3b-2c1acfbdd22c

- You should run all test for following folder and they should pass.
https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/folder/12519377-4b8e2d55-4e2d-4271-950e-befdd6ffa6cc
